combat-trainer.lic

```diff
@@ -359,7 +359,7 @@ class SetupProcess
     @cycle_regalia = nil
     Flags.reset('ct-regalia-expired')
     @equipment_manager.wear_equipment_set?('standard')
-    @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill) if @gearsets['standard'].include?(game_state.weapon_name) # The above will put away worn weapons
+    @equipment_manager.wield_weapon?(game_state.weapon_name, game_state.weapon_skill) if @gearsets['standard'].include?(game_state.weapon_name) # The above will put away worn weapons
```

Comment:
If it is easy, this would be a good candidate for a separate PR to keep this PR focused just on the one change. If it's not easy, don't worry about it.